src/card.css

```
@@ -0,0 +1,79 @@
+.card {
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  margin: 0 0 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: var(--text-dark-color);
+}
+
+.card__product {
+  margin-bottom: 16px;
+  color: var(--text-light-color);
+}
+
+.card__rating-review {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.card__review {
+  color: var(--text-dark-color);
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.card__price-text {
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__price-value {
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: bold;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 11px 0;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  color: #fff;
```

Create variables for all colors)
```suggestion
  color: var(--color-white);
```